docs/src/main/paradox/java/http/server-side/server-https-support.md

```diff
@@ -0,0 +1 @@
+../../../scala/http/server-side/server-https-support.md
```

Comment:
Looks like this is actually simple: we can just drop a `.htaccess` in `src/main/paradox` with something like:

```
RedirectMatch 301 ^(.*)/server-side-https-support.html$ $1/server-side/https-support.html
```

and that appears to work